contents/product/session-recording.mdx

```diff
@@ -35,6 +35,28 @@ documentation: /docs/user-guides/recordings
 
 <TutorialsSlider topic="session recordings" />
 
+### How does PostHog compare?
+
+|                                       | HotJar | Logrocket |  |   Matomo  |                              PostHog                              |
+|---------------------------------------|:------:|:---------:|:--------:|:---------:|:-----------------------------------------------------------------:|
+|                                                              **Platform**                                                                             |
+| Free plan                             |    ✅   |     ✅     |          |    ✅     |                                 ✅                               |
+| Open source                           |    ❌   |     ❌     |          |    ✅     |                                 ✅                               |
+| Self-hostable                         |    ❌   |     ✅     |          |    ✅     |                                 ✅                               |
+| Cloud hosting                         |    ✅   |     ✅     |          |    ✅     |                                 ✅                               |
+| Product analytics suite               |    ❌   |     ✅     |          |    ✅     |                                 ✅                               |
+|                                                              **Session Recording**                                                                    |
+| Single-page app support               |    ✅   |     ✅     |          |     ✅     |                                 ✅                              |
+| Mobile recordings                     |    ✅   |     ✅     |          |     ✅     |                                 ✅                              |
+| Identity detection                    |    ✅   |          |          |     ✅     |                                 ✅                                |
+| Target recordings by URL              |    ✅   |          |          |     ✅     |                                 ✅                                |
+| Target by sample size                 |    ❌   |          |          |     ✅     |                                 ✅                                |
+| Filter recordings by user or cohort   |    ❌   |          |          |     ✅     |                                 ✅                                |
+| Filter recordings by event            |        |          |          |           |                                 ✅                                 |
+| Privacy masking for sensitive content |    ✅   |     ✅     |          |     ✅     |                                 ✅                               |
+| Export session data                   |    ✅   |          |          |     ✅     |                                 ❌                                 |
+| Retention policy for recordings       | 1 year |    None   |      | 24 months | 3 weeks on PostHog Cloud<br/>Unlimited on self-hosted |
```

Comment:
I'd say "fully configurable" or something for us on self hosted. Unlimited feels kind of wrong.

Also, I think LogRocket has 1 month retention.

Reply:
I've changed to no limits. Fully configurable doesn't ring right to me either, as it doesn't really answer the topic.